src/routes/post.js

```diff
@@ -1,10 +1,11 @@
 import { Router } from 'express'
-import { create, getAll } from '../controllers/post.js'
+import { create, getAll, createComment } from '../controllers/post.js'
 import { validateAuthentication } from '../middleware/auth.js'
 
 const router = Router()
 
 router.post('/', validateAuthentication, create)
 router.get('/', validateAuthentication, getAll)
+router.post('/comment', validateAuthentication, createComment)
```

Comment:
Rather than send the postId in side the request body, it might better to have it as a route param? For example:

```javascript
router.post('/:id/comment', validateAuthentication, createComment)
```

You'll need to update your controller to get the post if from req.params, and also need to convert it to an int before passing to Prisma,